website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+1. Add the design system libraries to your SystemJS import map, then mark them as external ([full documentation](/docs/recommended-setup#sharing-with-import-maps)). Alternatively, do the equivalent with [module federation](/docs/recommended-setup#sharing-with-module-federation).
+1. Create a utility microfrontend (often called `@your-org-name/styleguide`) that contains all shared css and javascript components. Re-export the components from the design system so that all other microfrontends can access them via [cross microfrontend import](/docs/recommended-setup#cross-microfrontend-imports) (`import { Button } from '@your-org-name/styleguide';`).
+
+Once the design system is properly shared, all its CSS and javascript components will only be included one time on the web page. The code using the design system's components remains unchanged.
+
+### Global CSS versus shared Javascript components
+
+It's possible to share CSS via global CSS classes, Javascript components, or both. No method is clearly superior than the other in every way, and you should choose an approach that fits best how you'd like to do things.
+
+Some organizations scope the CSS for their shared Javascript components as a way of ensuring that the look and feel requires that you use the Javascript components. However, other organizations publish global css in addition to their javascript components, to allow for framework flexibility of their look and feel.
+
+To share Javascript components, use [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports).
+
+### CSS Variables
+
+Browsers support [CSS custom properties (CSS variables)](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), which facilitate sharing CSS between microfrontends in an easy way. Any CSS variable applied to the `:root` pseudoelement is accessible to any other microfrontend.
+
+```css
+/* In your styleguide / design system */
+:root {
+  --blue: #0000FF;
+}
+```
+
+```css
+/* In an individual microfrontend */
+.settings {
+  color: var(--blue);
+}
+```
+
+No extra configuration is needed for this to work, as this is built into the browser.
+
+## Scoped CSS
+
+For all CSS specific to a particular microfrontend or component, it is preferred to scope the CSS. In general, CSS classes are global by default, but "scoping" refers to encapsulating the CSS such that it only applies to one component or microfrontend. The code snippets below demonstrate some ways that this is possible:
+
+```css
+/*
+  GLOBAL: this css class is not scoped
+
+  <div class="settings"></div>
+*/
+.settings {
+  color: blue;
+}
+
+/*
+  Scoped by suffixing all css classes with a unique hash. This is often done by build tools,
+  particularly CSS Modules via webpack's css-loader (https://webpack.js.org/loaders/css-loader/).
```

Comment:
```suggestion
  particularly CSS Modules via Webpack's css-loader (https://webpack.js.org/loaders/css-loader/).
```

I also want to say something like `Webpack's css-loader loader` but that sounds weird.